tools/submission/submission-checker.py

```diff
@@ -982,28 +987,29 @@ def check_power_dir(power_path, ranging_path, testing_path, scenario_fixed, more
         is_valid = False
     else:
         avg_power = sum(power_list) / len(power_list)
+        power_duration = (power_end - power_begin).total_seconds()
         if scenario_fixed in ["Offline", "Server"]:
             # In Offline and Server scenarios, the power metric is in W.
             power_metric = avg_power
-        elif scenario_fixed in ["MultiStream"]:
-            # In SingleStream and MultiStream scenarios, the power metric is in J/sample.
-            power_duration = (power_end - power_begin).total_seconds()
-            num_samples = mlperf_log["generated_query_count"] * mlperf_log["generated_samples_per_query"]
-            power_metric = avg_power * power_duration / num_samples
-        elif scenario_fixed in ["SingleStream"]:
-            # TODO: Currently, LoadGen does NOT print out the actual number of queries in detail logs. There is a
-            # "generated_query_count", but LoadGen exits early when the min_duration has been met, so it is not equal to
-            # the actual number of queries. For now, we will make use of "result_qps_with_loadgen_overhead", which is
-            # defined as: (sample_count - 1) / pr.final_query_issued_time, where final_query_issued_time can be
-            # approximated by power_duration (off by one query worth of latency, which is in general negligible compared
-            # to 600-sec total runtime and can be offsetted by removing the "+1" when reconstructing the sample_count).
-            # Not an ideal approach, but probably the best we can do in v1.0.
-            # As for MultiStream, it always runs for 270336 queries, so using "generated_query_count" as above is fine.
-            # In v1.1, we should make LoadGen print out the actual number of queries and use it to compute the J/sample 
-            # for SingleStream and MultiStream power metrics.
-            power_duration = (power_end - power_begin).total_seconds()
-            num_samples = mlperf_log["result_qps_with_loadgen_overhead"] * power_duration
-            power_metric = avg_power * power_duration / num_samples
+        else:
+            # In SingleStream and MultiStream scenarios, the power metric is in J/query.
```

Comment:
@s-idgunji Now that for MultiStream, samples_per_query is always 8, what should be the power metric for MultiStream? Is it J/query or J/sample? (they will be 8x difference)

Reply:
I think J/sample should be easier on the eye to detect any anomalies. We expect efficiency to increase with increasing the number of samples per query, i.e. SS[J/sample] > MS[J/sample].

Reply:
@s-idgunji I'm not saying that dividing (or, equally, multiplying) by eight is hard to calculate. I'm saying that it's easier to perform a quick sanity check when SS and MS both use the J/sample metric. Moreover, when MS is inferred from SS, the energy number will be exactly the same, instantly signalling to the reader that the MS number is inferred, therefore, approximated.

Reply:
We discussed this in the Power WG meeting. The conclusion was that since we report the latency per query both for SS and MS, we should report the energy per query as well.